### ISSUE TITLE
IF ON AIR/GROUND EVENT SCRIPTS

### ISSUE THREAD

**Issue by ELV13Stm** [chrismaltby/gb-studio#696]
**Is your feature request related to a problem? Please describe.**
I find it odd that a platformer mode doesnt have this event yet.

**Describe the solution you'd like**
I was hoping something the lines of IF PLAYER IS ON AIR or GROUND event simple as that, I cant freaking describe this event enough.

**Comment by maxoakland** [comment]
Yeah, I would say this is pretty much mandatory. There are many situations where I could use this feature
